tests/knex-flex-filter.test.js

```diff
@@ -0,0 +1,283 @@
+
+import seedsFn from './helpers/seeds';
+import knex from './knex';
+import knexFlexFilter, { jsonbPreprocessor } from '../src';
+
+require('./helpers/database');
+
+
+describe('knex-flex-filter', () => {
+  let castFn;
+
+  beforeEach(async (done) => {
+    await seedsFn(knex);
+    castFn = (arg) => {
+      switch (arg) {
+        case 'ownerId':
+          return 'bigint';
+        case 'lastBuyBlockNumber':
+          return 'bigint';
+
+        default:
+          return undefined;
+      }
+    };
+    done();
+  });
+
+  describe('when filtering a normal column', () => {
+    it('correctly filters by _eq', async (done) => {
+      const query = knexFlexFilter(knex.table('entities'), { ownerId_eq: 1 }, { castFn });
```

Comment:
so this would work with objection using Model.query() just the same right=?

Reply:
Exactly 👍 should work with any query builder on top of knex.